pkg/utils/utils.go

```diff
@@ -95,25 +82,20 @@ func BeautifyMD(data []byte) (string, error) {
 }
 
 // ReadREADME: returns the byte array of README.md of a project
-func ReadREADME(projectName string) ([]byte, error) {
-	path, err := db.GetRecord(db.DBName, projectName, pkg.ProjectPaths)
-	if err != nil {
-		actualName, err := db.GetRecord(db.DBName, projectName, pkg.ProjectAliasBucket)
-		if err != nil {
-			log.Printf("project: %v not a valid project\n", projectName)
-			return nil, errors.Join(ErrReadREADME, err)
-		}
+func ReadREADME(dbname, projectName string) ([]byte, error) {
+	actualName, err := db.GetRecord(dbname, projectName, pkg.ProjectAliasBucket)
+	if err == nil {
 		projectName = actualName
-		path, err = db.GetRecord(db.DBName, projectName, pkg.ProjectPaths)
-		if err != nil {
-			log.Printf("project: %v not a valid project\n", projectName)
-			return nil, errors.Join(ErrReadREADME, err)
-		}
+	}
+	path, err := db.GetRecord(dbname, projectName, pkg.ProjectPaths)
+	if err != nil {
+		log.Printf("project: %v not a valid project\n", projectName)
+		return nil, errors.Join(ErrGetProject, err)
```

Comment:
Could you rebase from master and get the [updated implementation](https://github.com/theredditbandit/pman/blob/d36532d0c462b8c2451d5f0fa37fbae2bc7aee49/pkg/utils/utils.go#L98) of the `ReadREADME` function

Reply:
I did rebase with a few changes to the ReadREADME in order to be able to run the tests. more specifically, to separate the test DB from the real DB.

Reply:
I see , but this implementation is buggy. The problem is about how bbolt stores key value pairs. Initially while coding this it was my understanding that the we can retrieve a `value` using it's `key` but turns out it acts as a bi-directional dict so it is also possible to get a `key` by it's `value`. 

This causes a problem because , when I select any project in the interactive menu that has an alias like so 
![image](https://github.com/theredditbandit/pman/assets/85390033/39a8c05b-1508-4fd3-b802-bdf163cc71ee)

the name is displayed as `build-a-database (db)` so internally im first splitting it which gets me an array 
`[build-a-database , db]` and then I pass the first element of this array to the ReadREADME function, but because of bbolt works as a bidict now , the project name now retrieves the alias and the error is `nil`, which means the actualName variable now has `db` and since there is no project with the name db it crashes.

![image](https://github.com/theredditbandit/pman/assets/85390033/3eea6adc-29b3-4e83-95d1-89f581d31e41)

---

To remedy this problem , I have updated the implementation of the function

![image](https://github.com/theredditbandit/pman/assets/85390033/9b5ac357-19e3-4572-aa75-6c1e62dce34d)


I hope this makes sense , please let me know if there is any confusion.

Reply:
ooh! i see. so if i pass `dbname` as a parameter to `readREADME` function, i hope that won't change anything in the sense that it won't break the new logic?

Reply:
No, It shouldn't break anything as long as you also pass the `dbname` parameter wherever the `ReadREADME` function is also called